randovania/games/dread/json_data/Cataris.txt

```diff
@@ -1183,12 +1189,15 @@ Extra - asset_id: collision_camera_015
   > Tunnel to Double Obsydomithon Room
       Any of the following:
           All of the following:
-              Varia Suit
-              Spider Magnet or Simple IBJ or Use Spin Boost
+              Any of the following:
+                  Varia Suit
+                  Heat/Cold Runs (Intermediate) and Heat Damage ≥ 200
+              Any of the following:
+                  Spider Magnet or Space Jump
+                  Spin Boost and Walljump (Beginner)
           All of the following:
-              Heat/Cold Runs (Intermediate) and Heat Damage ≥ 200
-              Spider Magnet or Use Spin Boost
-          Heat/Cold Runs (Intermediate) and Heat Damage ≥ 300 and Simple IBJ
+              Simple IBJ
+              Varia Suit or Heat/Cold Runs (Intermediate) or Cold Damage ≥ 300
```

Comment:
Heat runs and [heat] damage need to be combined

Reply:
yeah this was sloppy on my end even for re-writing the whole formatting